src/devices/Mcp23xxx/Mcp23xxx.cs

```diff
@@ -4,75 +4,137 @@
 
 using System;
 using System.Device.Gpio;
-using System.Device.Spi;
 
 namespace Iot.Device.Mcp23xxx
 {
-    public class Mcp23xxx : IDisposable
+    // IGpioControllerProvider.  TODO: https://github.com/dotnet/iot/issues/125
```

Comment:
NIT: can you add a more verbose comment here on what this interface will have in order to avoid an extra click?

Reply:
Updated with comments.